app.rb

```diff
@@ -0,0 +1,153 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+@book_list = []
+@person_list = []
+@rental_list = []
+
+def message
+  p 'WELCOME TO OOP SCHOOL LIBRARY APP'
+  p 'Please Choose an option using a number.'
+  p '1 - List all books'
+  p '2 - List all people'
+  p '3 - Create a person'
+  p '4 - Create a book'
+  p '5 - Create a rental'
+  p '6 - List all rentals for a given person id.'
+  p '7 - Exit'
+end
+
+def list_books()
+  puts
+  if @book_list.empty?
+    print '\nPlease first add some book\n'
+  else
+    @book_list.each_with_index do |book, index|
+      puts "[#{index + 1}] Title : #{book.title}, Author: #{book.author}"
+    end
+  end
+end
+
+def list_person(persons)
+  persons.each do |person|
+    p "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+  end
+end
+
+def create_student()
+  p 'Add Student'
+  print('Name: ')
+  name = gets.chomp
+  print('Age: ')
+  age = gets.chomp
+  print('Has parent permission? [Y/N]: ')
+  parent_permission = gets.chomp
+  student = Student.new(name, age, parent_permission.upcase)
+  @person_list.push(student)
+  p 'Student created successfully\n'
+end
```

Comment:
- [ ] [REQUIRED] While I try to create a student and I try to specify if he/she has parent permission, I get this error. Kindly fix it.
<img width="907" alt="Screenshot 2022-11-19 at 18 21 45" src="https://user-images.githubusercontent.com/58692590/202863533-4ecee303-a5c1-43cb-94d9-b020fef99410.png">